libraries/AP_HAL_ChibiOS/hwdef/PixPilot-V6PRO/hwdef-bl.dat

```diff
@@ -0,0 +1,62 @@
+# hw definition file for processing by chibios_hwdef.py
+# for H743 bootloader
+
+# MCU class and specific type
+MCU STM32H7xx STM32H743xx
+
+# crystal frequency
+OSCILLATOR_HZ 24000000
+
+# board ID for firmware load
+APJ_BOARD_ID 1160
+
+FLASH_SIZE_KB 2048
+
+# bootloader is installed at zero offset
+FLASH_RESERVE_START_KB 0
+
+# the location where the bootloader will put the firmware
+# the H743 has 128k sectors
+FLASH_BOOTLOADER_LOAD_KB 128
+
+env OPTIMIZE -Os
+PB3 LED_ACTIVITY OUTPUT OPENDRAIN HIGH # red
+PE12 LED_BOOTLOADER OUTPUT OPENDRAIN HIGH # blue
+define HAL_LED_ON 0
```

Comment:
remove... will now cause build error

Reply:
Yes, if I remove it, it will build error. Now it works fine.